docker-compose.yml

```diff
@@ -58,6 +61,104 @@ services:
     - volume
     - consul
 
+  vault:
+    image: edgexfoundry/docker-edgex-vault:0.2.0
```

Comment:
Need to parameterize this and pull the image from nexus3.edgexfoundry.org:10004, which is the staging repo.
The parameters are conditionally set by `bin/env.sh` and `bin/arm64_env.sh`.  Not the docker image names noting arm64 available in nexus.
This way we can be sure that we are always testing the latest image available for both x86 and arm architecture.